Hashtable.h

```diff
@@ -9,9 +9,10 @@ in the source distribution for its full text.
 
 #include <stdbool.h>
 #include <stddef.h>
+#include <stdint.h>
 
 
-typedef unsigned int ht_key_t;
```

Comment:
Are you *sure* you can do that? Note that Hashtable is used in quite a few places in the codebase, and [at least one](https://github.com/htop-dev/htop/blob/0e29174211754774534a5163667a91dbaa84abee/linux/LinuxProcessList.c#L696-L700) uses it to store an inode number, which can be 64-bit at least on 64-bit systems (on 32-bit ones that seems to depend on `_FILE_OFFSET_BITS=64`).

Reply:
Would be nice if this would be either left in it's original state or updated to make key space conversions/hashing explicit as elaborated in [my comment from last year](https://github.com/htop-dev/htop/pull/998/files#r886543390) …